ts/javascript/mod.test.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  afterEach,
+  beforeEach,
+  describe,
+  it,
+} from "https://deno.land/std@0.206.0/testing/bdd.ts";
+import * as garn from "./mod.ts";
+import { buildPackage } from "../testUtils.ts";
+import { assert } from "https://deno.land/std@0.206.0/assert/mod.ts";
+import { existsSync } from "https://deno.land/std@0.201.0/fs/exists.ts";
+
+describe("mkNpmProject", () => {
+  let tempDir: string;
+  beforeEach(() => {
+    tempDir = Deno.makeTempDirSync({ prefix: "garn-test" });
+  });
+  afterEach(() => {
+    Deno.removeSync(tempDir, { recursive: true });
+  });
+  it("produces correct store path structure", () => {
+    const packageJson = `
+      {
+        "name": "my-npm-project",
+        "version": "1.0.0",
+        "description": "My npm project",
+        "dependencies": {
+          "typescript": "*"
+        }
+      }
+    `;
```

Comment:
I think I would declare the contents of these functions as objects in typescript and then write them out with `JSON.stringify`. That way we get code formatting any syntax highlighting for the data. Also, I would inline `packageJson`. Like this:

```suggestion
    Deno.writeTextFileSync(`${tempDir}/package.json`, {
      "name": "my-npm-project",
      "version": "1.0.0",
      "description": "My npm project",
      "dependencies": {
        "typescript": "*"
      }
    });
    `;
```

(Same for `package-lock.json`.)